dev/breeze/src/airflow_breeze/shell/enter_shell.py

```diff
@@ -206,3 +206,31 @@ def enter_shell(**kwargs):
         console.print(CHEATSHEET, style=CHEATSHEET_STYLE)
     enter_shell_params = ShellParams(**filter_out_none(**updated_kwargs))
     run_shell_with_build_image_checks(verbose, dry_run, enter_shell_params)
+
+
+def find_airflow_container(**kwargs):
```

Comment:
Better to pass the `verbose` and `dry_run` explicitly as parameters. no need to go via kwargs.